improver_tests/threshold/test_BasicThreshold.py

```diff
@@ -449,6 +449,16 @@ def test_threshold_unit_conversion(self):
         result = plugin(self.rate_cube)
         self.assertArrayAlmostEqual(result.data, expected_result_array)
 
+    def test_threshold_unit_conversion_2(self):
+        """Test threshold coordinate points after unit conversion of small
+        numbers."""
```

Comment:
Maybe `Test threshold coordinate points precisely after unit conversion of small floating point values with no tolerance for floating point precision errors.`? That or something similar might help to highlight the fact that you're intentionally using `assertArrayEqual` here, rather than `assertArrayAlmostEqual`, which might be what is expected for floating point values.